src/components/project/dto/create-project.dto.ts

```diff
@@ -27,6 +28,9 @@ export abstract class CreateProject {
 
   @DateField({ nullable: true })
   readonly estimatedSubmission?: CalendarDate;
+
+  @Field(() => ProjectStep, { nullable: true })
+  readonly step?: ProjectStep = ProjectStep.EarlyConversations;
```

Comment:
```suggestion
  readonly step: ProjectStep = ProjectStep.EarlyConversations;
```
This will fix the need to not-null assert below.

Reply:
Ah ok I actually had that at one point but didn't know if that's ok